frontend/packages/console-app/src/components/quick-starts/QuickStartDrawer.tsx

```diff
@@ -37,19 +36,25 @@ type DispatchProps = {
   onClose: () => void;
 };
 
-type QuickStartDrawerProps = StateProps & DispatchProps;
+type QuickStartDrawerProps = StateProps &
+  DispatchProps & {
+    quickStarts: QuickStart[];
+  };
 
 const QuickStartDrawer: React.FC<QuickStartDrawerProps> = ({
   children,
   activeQuickStartID,
   activeQuickStartStatus,
   onClose,
+  quickStarts,
 }) => {
   const [modalOpen, setModalOpen] = React.useState<boolean>(false);
-  const quickStart = getQuickStartByName(activeQuickStartID);
   const [scrollDirection, handleScrollCallback] = useScrollDirection();
   const { t } = useTranslation();
 
+  const index = quickStarts.findIndex((qs) => qs.metadata.name === activeQuickStartID);
+  const quickStart = quickStarts[index];
```

Comment:
```suggestion
  const quickStart =  quickStarts.find((qs) => qs.metadata.name === activeQuickStartID);
```

Reply:
Also wrap this in `useMemo`

Reply:
updated, not using Memo as quickStarts now moved to props.